tests/UtestPlatformTest.cpp

```diff
@@ -30,6 +30,31 @@
 #include "CppUTest/TestTestingFixture.h"
 #include "CppUTest/PlatformSpecificFunctions.h"
 
+TEST_GROUP(UTestPlatformsTest)
+{
+};
+
+TEST(UTestPlatformsTest, PlatformSpecificFileFunctionsDontCrash)
+{
+    PlatformSpecificFile file;
+    file = PlatformSpecificFOpen("/dev/null", "rw");
+    PlatformSpecificFPuts("Hello", file);
+    PlatformSpecificFClose(file);
+}
+
+TEST(UTestPlatformsTest, GetPlatformSpecificTimeStringIsCovered)
+{
+    GetPlatformSpecificTimeString();
```

Comment:
I'd prefer changing it so it gets passed seconds and can be genuinely tested.

Also, you forgot about option #3: All of these functions can be removed entirely. They are dead code; not used anywhere.